Setup.hs

```diff
@@ -1,3 +0,0 @@
-import Distribution.Simple
-main = defaultMain
-
```

Comment:
I don't know enough about this; are these files for older Cabal versions than we support?